app/(setting)/delete/page.tsx

```
@@ -0,0 +1,59 @@
+import { createClient } from '@/lib/supabase/client';
+import styles from './page.module.scss';
+
+const DeletePage = () => {
+  const router = useRouter();
+  // TODO style gap 관련하여 dvh 처리? 고민
+  const [isLoading, setIsLoading] = useState(false);
+
+
+
+  const handleDelete = async () => {
+    if (!confirm('정말 탈퇴하시겠습니까?')) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)